content/2018-03-14-agradecimentos-campanha-pybr13.md

```diff
@@ -0,0 +1,266 @@
+
+Em sua 13ª edição, este ano a PyBR teve 98 mulheres inscritas, o que resulta em 17% dos participantes da conferência. Esse número ainda é pequeno, mas já mostra uma evolução no cenário em comparação com as edições anteriores, e que há coisas sendo feitas para mudar a realidade da inclusão de mulheres na comunidade Python. Uma delas é justamente a participação massiva da comunidade na nossa campanha, que deu muito certo!
+Outro dado que chamou nossa atenção foi que 60% do público da PyBR13 estava em sua primeira participação na conferência. A comunidade Python também se esforça para abraçar tanto experts quanto iniciantes, e é uma enorme satisfação receber cada vez mais pessoas interessadas na nossa linguagem de programação preferida. Essa é uma das razões pelas quais queremos participar todo ano desse evento maravilhoso! <3
+
+O principal objetivo do PyLadies é promover a diversidade no setor de tecnologia, incentivando mulheres a tornarem-se participantes ativas e líderes na comunidade Python. Para isso, é fundamental a inclusão de mulheres em eventos como a Python Brasil, e por isso todos os anos organizamos uma campanha de financiamento coletivo para prover auxílio às ladies que de outro modo não teriam condições de participar da conferência. Este ano conseguimos levar *todas* as participantes que precisavam do auxílio, graças a todas as pessoas que contribuíram com nossa campanha fazendo doações de qualquer valor. Muito além das recompensas que preparamos para quem contribuiu, como camisetas, canecas e adesivos, o mais importante que temos a lhes dedicar é nosso **muito obrigada!**
+
+Algumas mensagens das beneficiadas pela campanha:
+
+<style>
+@import url(https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css);
+@import url(https://fonts.googleapis.com/css?family=Raleway:400,800);
+figure.quote {
+  font-family: 'Raleway', Arial, sans-serif;
+  position: relative;
+  float: left;
+  margin: 10px 1% 40px;
+  min-width: 400px;
+  max-width: 600px;
+  width: 100%;
+  color: #333;
+  text-align: left;
+}
+figure.quote * {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+figure.quote img {
+  float: left;
+  max-width: 30%;
+  vertical-align: middle;
+}
+figure.quote figcaption {
+  float: left;
+  max-width: 70%;
+  background-color: #ffffff;
+}
+figure.quote blockquote {
+  position: relative;
+  padding: 5px 25px 5px 25px;
+  font-size: 0.8em;
+  font-weight: 500;
+  text-align: left;
+  margin: 0;
+  line-height: 1.6em;
+  font-style: italic;
+}
+figure.quote h5 {
+  opacity: 0.8;
+  font-weight: 800;
+  margin: 5px 20px;
+  color: #212121;
+  text-transform: uppercase;
+}
+figure.quote h5 span {
+  font-weight: 400;
+  text-transform: none;
+  padding-left: 5px;
+}
+body {
+  background-color: #ffffff;
+  display: inline-block;
+}
+</style>
+<html>
+<head><meta charset="UTF-8"></head>
+<body>
```

Comment:
Pode remover esse
```
<html>
<head><meta charset="UTF-8"></head>
<body>
```
Pois todo esse conteúdo já ficará dentro dessas tags quando for colocado no template (https://github.com/pyladies-brazil/br-pyladies-pelican/blob/develop/themes/default/templates/article.html#L40)